Add sphinx docs

jbradberry opened the pull request:
TEQ-1

jbradberry committed:
Fill in the developer docs for the script

dchukhin commented on README.rst:
```diff
@@ -23,24 +23,26 @@ with either Python 2.7 or Python 3.
+For development purposes, create a virtualenv, then clone the repo to
 your local machine and pip install it ::
 
     $ mkvirtualenv tequila -p $(which python3)
```
Perhaps we should call this virtualenv tequila-cli?

jbradberry committed:
Update the README.rst to link back to caktus/tequila

and to include information about setting up tequila-cli for
development purposes.

dchukhin reviewed (approved):
🆗
